README.md

```diff
@@ -34,13 +35,12 @@ Best practices for using the Office.js NPM package include:
 
 ### Support
 
-The Office.js CDN is the official, supported source for Office Add-ins. This means that for the NPM package the following are not guaranteed.
-
-- Frequency of updates
+The Office.js CDN is the official, supported source for Office Add-ins. For the NPM package sourced through this repository, only the latest version of the package is supported. We'll not support nor apply any patches to previous major/minor versions of the package. In addition, the following are guaranteed for the NPM package: 
 
-- Timely support for issues and bugs
+- Turnaround time for resolution of issues and bugs. 
```

Comment:
What kind of turnaround time?

Reply:
Turnaround time - time to fix bugs and issues. We don’t know how long it’ll take to fix issues down to NPM level.